manifest.json

```diff
@@ -550,6 +550,16 @@
                             "sha256": "f846d313facedc1115da8ff492c56ab9cf152d8eedc017f3334cfb8128807f47"
                         }
                     ]
+                },
+                "1.0.1": {
+                    "releaseUrl": "https://github.com/Nytra/NeosAudioClipWizard/releases/tag/v1.0.1",
+                    "changelog": "Small code tweaks.",
```

Comment:
changelogs aren't required for each version, if you don't have much to say for a given change log, you can exclude it entirely